src/ensembl/src/shared/components/error-boundary/ErrorBoundary.test.tsx

```diff
@@ -15,62 +15,82 @@
  */
 
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
+import noop from 'lodash/noop';
 
 import errorService from 'src/services/error-service';
 
 import ErrorBoundary from './ErrorBoundary';
 
-const Child = () => <span>I am a child</span>;
+const Child = () => <span className="child">I am a child</span>;
+const BrokenChild = (props: { errorMessage: string }) => {
+  throw new Error(props.errorMessage);
+
+  return <div className="child">You shouldn't see me!</div>;
+};
 
 const Fallback = ({ error }: { error: Error }) => (
-  <span>I am the fallback component that received error {error}</span>
+  <span className="fallback">
+    I am the fallback component that received error {error.message}
+  </span>
 );
 
 jest.spyOn(errorService, 'report');
```

Comment:
Can we use `jest.spyOn(errorService, 'report').mockImplementation(noop);` here and remove the `beforeEach` block on L47?

Reply:
I don't want errorService to execute its real report method during any of the tests (otherwise it will spam the console with the error message). What I can do is I can replace the `jest.spyOn(errorService, 'report').mockImplementation(noop);` with `errorService.report = jest.fn()` and put it in the `beforeAll` function, so that it runs once per the whole describe block.